R/convert_data.R

```diff
@@ -41,15 +41,16 @@ spod_convert <- function(
   quiet = FALSE,
   max_mem_gb = max(4, spod_available_ram() - 4),
   max_n_cpu = parallelly::availableCores() - 1,
-  max_download_size_gb = 1
+  max_download_size_gb = 1,
+  ignore_missing_dates = FALSE
```

Comment:
👍 error is a good default.

Reply:
thanks for catching this!

Reply:
What did I catch?

Reply:
ahh, i though I wrongly defaulted it to TRUE. I misinterpreted this comment as a comment to your commit correcting me) Anyway, yes, good to error by default, as this is extremely important for aggregations that are dependent on number of non-missing days in a month